util/fpga/README.md

```diff
@@ -20,7 +20,7 @@ Updated output bitfile located : at the same place as raw vivado bitfile @
 
 This directory contains following files:
 
-* `splice_nexysvideo.sh` - master script
+* `splice_nexysvideo.sh` - over-arching script
```

Comment:
IMHO "main script" is clearer, but I don't feel particularly strongly on this

Reply:
Or maybe this is an opportunity to convey more information. How about `script to load memory contents into bitfile` (nicked and paraphrased from the top of that file)?

Reply:
sg fixing